tests/loki/iofq_test.py

```diff
@@ -102,16 +108,17 @@ def test_pipeline_can_compute_IofQ_in_overlapping_wavelength_bands():
     ).transpose()
     pipeline = sciline.Pipeline(loki_providers(), params=params)
     result = pipeline.compute(BackgroundSubtractedIofQ)
-    assert result.dims == ('band', 'Q')
+    assert result.dims == ('band', 'Qy', 'Qx') if qxy else ('band', 'Q')
     assert result.sizes['band'] == 10
 
 
-def test_pipeline_can_compute_IofQ_in_layers():
-    params = make_params()
+@pytest.mark.parametrize('qxy', [False, True])
+def test_pipeline_can_compute_IofQ_in_layers(qxy: bool):
+    params = make_params(qxy=qxy)
     params[DimsToKeep] = ['layer']
     pipeline = sciline.Pipeline(loki_providers(), params=params)
     result = pipeline.compute(BackgroundSubtractedIofQ)
-    assert result.dims == ('layer', 'Q')
+    assert result.dims == ('band', 'Qy', 'Qx') if qxy else ('band', 'Q')
```

Comment:
I don't understand why `layer` has changed to `band`.

Reply:
Copy and paste error from above :grimacing: